charts/sourcegraph/templates/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -45,6 +45,26 @@ spec:
         app: sourcegraph-frontend
         deploy: sourcegraph
     spec:
+      initContainers:
+      {{- if .Values.migrator.enabled }}
+      - name: migrator
+        image: {{ include "sourcegraph.image" (list . "migrator") }}
+        args:
+        - up
+        env:
+        {{- range $name, $item := .Values.frontend.env }}
+        - name: {{ $name }}
+          {{- $item | toYaml | nindent 10 }}
+        {{- end }}
+        {{- range $name, $item := .Values.migrator.env }}
+        - name: {{ $name }}
+          {{- $item | toYaml | nindent 10 }}
+        {{- end }}
```

Comment:
I am debating whether reusing all `frontend` env var is a good approach, thoughts?

Reply:
I like it! Agreed that they're not all required, but it's nice not to have to configure anything extra.